### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by MergenStudios**
Added windows support and changed the formatting to f-strings for easier readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Python and system information command-line tool with pure Python. Neofetch like
 - Shows compile and compiler version
 - Show kernel and operation system
 - Also light and dark color pallate
-- Only GNU/Linux
+- Only Linux/Windows
```
Comment (beucismis) [inline README.md:22]: Change to "Only GNU/Linux and Windows"

Comment (MergenStudios) [reply README.md]: changed it

**Comment by beucismis** [comment]
Hmm, this is interesting.
```sh
$ pyfetch 
Traceback (most recent call last):
  File "/home/adil/Clones/pythonfetch/.venv/bin/pyfetch", line 33, in <module>
    sys.exit(load_entry_point('pythonfetch==0.13.0', 'console_scripts', 'pyfetch')())
  File "/home/adil/Clones/pythonfetch/.venv/lib/python3.9/site-packages/pythonfetch-0.13.0-py3.9.egg/pythonfetch/__init__.py", line 101, in main
AttributeError: 'posix.uname_result' object has no attribute 'machiene'
```

File: pythonfetch/__init__.py [hunk 2]
```diff
@@ -63,36 +64,54 @@ def red(text):
...
+    elif os_type == "Linux":
+
+        uname = os.uname()
+        # https://docs.python.org/3/library/getpass.html#getpass.getuser
+        userinfo = f"{red(getpass.getuser())}@{red(uname.nodename)}"
+        splitline = (len(getpass.getuser()) + len(uname.nodename) + 1) * "─"
+
+        # information on the os
+        kernel = f"{red('Kernel')}: {uname.sysname}-{uname.release}"
+        os_info = f"{get_linux_os_name()} {uname.machiene}" if get_linux_os_name() != str() or None else SPACE
```
Comment (beucismis) [inline pythonfetch/__init__.py:101]: Change to "uname.machine"

Comment (MergenStudios) [reply pythonfetch/__init__.py]: fixed

**Comment by beucismis** [comment]
That looks good, @MergenStudios thank you.

**Pull request merged**